src/App.tsx

```diff
@@ -1,10 +1,14 @@
 import "./App.css";
+import Practice3 from "./templates/Practice3";
 
 function App() {
 	return (
-		<header className="bg-[#94A3B8] text-center p-[20px] text-4xl text-[#F9FAFB]">
+		<div className="box-border pb-80 mx-60 mt-20 h-200 w-104..."> 
+		<div className="bg-slate-400 text-center p-[20px] text-4xl text-gray-50...">
```

Comment:
Please remove ... and you can use `p-5` for 20 px

Reply:
Sorry the header was already there in the template we cloned, so didn't notice this. Have changed it now !